.github/workflows/prov-compat-label.yml

```diff
@@ -202,12 +201,15 @@ jobs:
         # Note that releases are not used as a test environment for
         # later providers.  Problems in these situations ought to be
         # caught by cross branch testing before the release.
-        tree_a: [ branch-3.5, branch-3.4, branch-3.3, branch-3.2, branch-3.0,
+        tree_a: [ branch-master, branch-3.6,
```

Comment:
I'd suggest making this a separate commit since it's unrelated to the main part of this work.  It could even be a separate PR.

Not a big deal if you don't want to change it.